edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/Microsoft.Azure.Devices.Edge.Agent.Diagnostics.csproj

```diff
@@ -1,20 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\dotnetVersion.props" />
+
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Why change to use .NetCore instead of .Net Standard?  I think it is ok to change, please confirm with Varun.

Reply:
The logic here is so that it can be built with any dotnet version that implements netstandard 2.1, and not just dotnet 3.1, correct?

Reply:
I tried to use netstandard, and found this issue which instructs us to target netcoreapp directly:
https://github.com/dotnet/sdk/issues/3667

Reply:
same comment as above, that seems to only apply to projects using asp.net, but I think most of the libraries should still work